src/main/resources/entity-types/finance/drv_fund_details.json5

```diff
@@ -0,0 +1,286 @@
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'metadata' ->> 'createdDate'"
+    },
+    {
+      name: 'updated_date',
+      dataType: {
+        dataType: 'dateType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'metadata' ->> 'updatedDate'"
+    },
+    {
+      name: 'created_by_user_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'metadata' ->> 'createdByUserId'"
+    },
+    {
+      name: 'updated_by_user_id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'metadata' ->> 'updatedByUserId'"
+    },
+    {
+      name: 'created_by_username',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'metadata' ->> 'createdByUsername'"
+    },
+    {
+      name: 'updated_by_username',
+      dataType: {
+        dataType: 'stringType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'metadata' ->> 'updatedByUsername'"
+    },
+    {
+      name: 'tag_list',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'stringType',
+        },
+      },
+      queryable: false,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb -> 'tags' ->> 'tagList'"
+    },
+    {
+      name: 'locations',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'objectType',
+          properties: [
+            {
+              name: 'location_id',
+              property: 'locationId',
+              dataType: {
+                dataType: 'rangedUUIDType',
+              },
+              queryable: false,
+              valueGetter: "( SELECT array_agg(elems.value ->> 'locationId') FROM jsonb_array_elements(fnd.jsonb -> 'locations') AS elems)",
+              filterValueGetter: "( SELECT array_agg(lower(elems.value ->> 'locationId')) FROM jsonb_array_elements(fnd.jsonb -> 'locations') AS elems)",
+              valueFunction: 'lower(:value)',
+            },
+            {
+              name: 'tenant_id',
+              property: 'tenantId',
+              dataType: {
+                dataType: 'rangedUUIDType',
+              },
+              queryable: false,
+              valueGetter: "( SELECT array_agg(elems.value ->> 'tenantId') FROM jsonb_array_elements(fnd.jsonb -> 'locations') AS elems)",
+              filterValueGetter: "( SELECT array_agg(lower(elems.value ->> 'tenantId')) FROM jsonb_array_elements(fnd.jsonb -> 'locations') AS elems)",
+              valueFunction: 'lower(:value)',
+            }
+          ],
+        },
+      },
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb ->> 'locations'",
```

Comment:
Should this be a `->` instead?

Reply:
@mweaver-ebsco  this field is same as POL fundDistribution. So everything is according to that (since it works in the expected way)